packages/block-library/src/navigation-menu/index.php

```diff
@@ -80,27 +80,38 @@ function render_block_navigation_menu( $attributes, $content, $block ) {
 
 	$colors = build_css_colors( $attributes );
 
-	return "<nav class='wp-block-navigation-menu' {$comp_inline_styles}>" .
-		build_navigation_menu_html( $block, $colors ) .
+	$css_classes = "wp-block-navigation-menu";
+
+	if ( ! empty( $colors['bg_css_classes'] ) ) {
+		$css_classes .= ' ' .  $colors['bg_css_classes'];
+	}
+
+	if ( ! empty( $colors['text_css_classes'] ) ) {
+		$css_classes .= ' ' .  $colors['text_css_classes'];
+	}
+
+	return sprintf(
+		'<nav class="%1$s" %2$s>%3$s</nav>',
+		esc_attr( $css_classes ),
+		$comp_inline_styles, // This should really be split out into attribute name and value to work like class above.
+		build_navigation_menu_html( $block )
+	);
+		build_navigation_menu_html( $block ) .
 	'</nav>';
```

Comment:
This will need another revision, sorry @retrofox 😬
I wasn't able to suggest replacing three lines, so it only replaced the first one. Although, coincidentally, this shouldn't even introduce a fatal error. Yay PHP!

Reply:
I was experimenting with this feature since I didn't know it was possible. Really nice, maybe it needs better handling, though. :-D